sql/sql_yacc.yy

```diff
@@ -6815,6 +6849,13 @@ attribute:
             Lex->type|=
               (COLUMN_FORMAT_TYPE_DYNAMIC << FIELD_FLAGS_COLUMN_FORMAT);
           }
+        | COLUMN_FORMAT_SYM COMPRESSED_SYM opt_with_compression_dictionary
```

Comment:
Hmm. It looks like they are.
However, this behavior existed even before I added 'compressed columns'.
For instance,
```sql
CREATE TABLE t(
  id INT,
  a BLOB COLUMN_FORMAT FIXED COLUMN_FORMAT DYNAMIC
) ENGINE=InnoDB;

SHOW CREATE TABLE t;
```
does not trigger any error and the most recent attribute is used
```sql
SHOW CREATE TABLE t;
Table   Create Table
t       CREATE TABLE `t` (
  `id` int(11) DEFAULT NULL,
  `a` blob /*!50606 COLUMN_FORMAT DYNAMIC */
) ENGINE=InnoDB DEFAULT CHARSET=latin1
DROP TABLE t;
```